### DISCUSSION TITLE
Redirecting user to Google Maps website

### DISCUSSION THREAD

**Discussion by abhishekmardiya** [visgl/react-google-maps#238] [General]
Is there an API available for redirecting the user to the Google Maps website and pinning the location provided by the user?

**Comment by abhishekmardiya** [comment]
![Screenshot (444)](https://github.com/visgl/react-google-maps/assets/97448460/e8e3356e-5bc3-4e18-9583-0b5602d69f81)


Like google iframe has an option to "view in larger map"

**Comment by usefulthink** [comment]
There's a specification for Maps URLs that can do all those things: https://developers.google.com/maps/documentation/urls/get-started

**Reply by abhishekmardiya** [reply]
Thank you!
